closure/closure-compiler/src/com/google/javascript/jscomp/JsMessage.java

```diff
@@ -164,7 +164,7 @@ public String getDesc() {
    * Gets the meaning annotated to the message, intended to force different
    * translations.
    */
-  String getMeaning() {
+  public String getMeaning() {
```

Comment:
Ouch! ;)

I'll do anything that needs to get done to contribute this patch (and bogart all the credit!).
I guess I need to get this line upstream before this can be accepted, is that the case?
Thanks.